code/game/objects/items/crayons.dm

```diff
@@ -682,7 +682,10 @@
 				else
 					target.set_opacity(initial(target.opacity))
 
-		. = use_charges(user, 2)
+		var/charges_used = use_charges(user, 2)
+		. = charges_used
+		if(!charges_used)
+			return FALSE
```

Comment:
```suggestion
		if(!.)
			return FALSE
```

Reply:
Both work, but using . in ifs is kind of eh but I mean it does save a useless variable so I am kind of mixed.